src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/FandomPageObject.java

```diff
@@ -0,0 +1,13 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class FandomPageObject extends WikiBasePageObject {
+
+  @FindBy(css = ".article-header__video")
+  private WebElement featuredVideo;
+
+  public boolean isFeaturedVideo() { return featuredVideo.isDisplayed(); }
```

Comment:
`wait.forElementVisible()` should be used before

Reply:
done